### PR TITLE
ghauri initial pr

### PR DESCRIPTION
**Pull request by dualfade**
please review; great new tool under rapid dev.

### CODE REVIEW

File: packages/ghauri/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+depends=('python' 'python-chardet' 'python-tldextract' 'python-colorama'
+         'python-requests')
+makedepends=('git' 'python-setuptools')
+source=("git+https://github.com/r0oth3x49/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd "$pkgname"
+
+  echo "$(git rev-list --count HEAD)"."$(git rev-parse --short HEAD)"
```
Comment (noraj) [inline packages/ghauri/PKGBUILD:21]: Use `git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'` please, there are tags available.

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/ghauri/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/ghauri/PKGBUILD:21]: I will adjust. Thanks.

**Comment by noptrix** [comment]
@dualfade we don't need to use quotes in `cd "$pkgname"`, btw. only when you "strcat()" it, e.g.: `"$pkgname-blah"` or `"$pkgname-$pkgver"`, etc. :) i am gonna remove the quotes here.

**Comment by dualfade** [comment]
> @dualfade we don't need to use quotes in `cd "$pkgname"`, btw. only when you "strcat()" it, e.g.: `"$pkgname-blah"` or `"$pkgname-$pkgver"`, etc. :) i am gonna remove the quotes here.

I only did so because the latest shellcheck was insisting ;)  got it. Appreciate it.

**Pull request merged**